### PR TITLE
Edit pass on additional considerations

### PR DESCRIPTION
**Pull request by ckittel**
Tried to group items on this Additional Considerations page a bit more.  Let me know what you think.

Quick link to view: <https://github.com/mspnp/aks-baseline-regulated/blob/additional-considerations-cleanup/docs/additional-considerations.md>

* Also added Priyanka's placeholder art for the cluster network.  Removed the warning since it's accurate now.  Not final art, but is our operating art until the final is worked on with the RA.
* Changed the main readme h1 based on CSU feedback

### CODE REVIEW

**Review by ferantivero — APPROVED** [review]
nice additions, specially the new diagram; only one comment to be reviewed

LGTM 🚀

File: docs/additional-considerations.md [hunk 1]
```diff
@@ -20,103 +23,190 @@ You can take OS and data disk encryption one step further and also bring the enc
...
+
+### Azure Key Vault network restrictions
+
+In this reference implementation, Azure Application Gateway (AAG) is sourcing its public-facing certificate from Azure Key Vault. This is great as it help support easier certificate rotation and certificate control. However, currently Azure Application Gateway only supports this on Azure Key Vault instances that are exclusively network restricted via Private Link. This reference implementation deploys Azure Key Vault in a hybrid model, supporting private link and public access specifically to allow AAG integration. Once [Azure Application Gateway supports private link access to Key Vault](https://docs.microsoft.com/azure/application-gateway/key-vault-certs#how-integration-works), we'll update this reference implementation. If this topology will not be suitable for your deployment, change the certificate management process in AAG to abandon the use of Key Vault for the public-facing TLS certificate and [handle the management of that certificate directly within AAG](https://docs.microsoft.com/azure/application-gateway/tutorial-ssl-cli). Doing so will allow your Key Vault instance to be fully isolated.
```
[the remark this answers is not in the document]
Comment (ckittel) [reply docs/additional-considerations.md:47]: whoops!  Great catch -- I worded that like a lie :P

[the remark this answers is not in the document]
Comment (ckittel) [reply docs/additional-considerations.md]: Updated in https://github.com/mspnp/aks-baseline-regulated/pull/4/commits/4298c8cad9fd4fe1f349e67303feded8ba896250

[the remark this answers is not in the document]
Comment (ckittel) [reply docs/additional-considerations.md]: Resolving, feel free to reopen if there is more.

**Pull request merged**